.github/workflows/openshift-upgrade-test.yaml

```diff
@@ -33,7 +33,13 @@ jobs:
   e2e-tests:
     name: Prepare E2E configuration and image
     runs-on: ubuntu-latest
+    environment: test
+    if: env.SKIP_OPENSHIFT != 'true'
```

Comment:
Lets try without the if and see the debug info
```suggestion
```

Reply:
I suspect the problem we might have here is that this ìf`chere does not have access to secrets or environment, it only can see GitHub events and things like that. Not sure though.